components/DiscordBar.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useEffect, useState } from 'react'
+import styles from '../scss/discordBar.module.scss'
+type Member = {
+  id: string
+  username: string
+  status: string
+  avatar_url: string
+}
+interface discordJSON {
+  members?: Member[]
+}
+const ONE_MINUTE = 60 * 1000
+const Member: React.FC<{ avatar: string }> = ({ avatar }) => {
+  return (
+    <div className="d-inline-block">
+      <img src={avatar} className={`${styles['member-avatar']} mx-1`} />
+    </div>
+  )
+}
+const DiscordBar: React.FC = () => {
+  interface State extends discordJSON {
+    error?: string
+  }
```

Comment:
Put types and interfaces outside the components scope.

Reply:
Done.